src/Components/Notifications/NotificationsList.tsx

```diff
@@ -199,7 +199,9 @@ export default function ResultList() {
           }
           className="relative py-5 px-4 lg:px-8 hover:bg-gray-200 focus:bg-gray-200 transition ease-in-out duration-150 cursor-pointer"
         >
-          <div className="text-lg font-bold">{result.event}</div>
+          <div className="text-lg font-bold">
+            {result.event.split("_").join(" ")}
```

Comment:
Use the notification type from constants 

https://github.com/coronasafe/care_fe/blob/036057a6e674b56dcf17aa6d85ee8743691e2520/src/Common/constants.tsx#L422

Reply:
done